### PR TITLE
Add evalcast-compliant variant of zyzzyva

### PR DESCRIPTION
**Pull request by sgsmob**
A version of the `zyzzyva` forecaster that conforms to the specifications of and can be used with the `evalcast` package.

The underlying forecast methodology is unchanged in this PR.

### CODE REVIEW

File: forecasters/zyzzyva/DESCRIPTION [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+           family = "Narasimhan",	
+           role = "ctb"))
+Description: The zyzzyva forecaster.
+License: MIT + file LICENSE
+Depends: R(>= 3.5.0)
+Encoding: UTF-8
+Suggests: testthat
+Imports:
+    forecast,
+    evalforecast,
```
Comment (dajmcdon) [inline forecasters/zyzzyva/DESCRIPTION:33]: I think this is just for `evalforecast::get_target_period()` right? I copied it into `evalcast`. Can you switch that (currently still needs the `killcards` branch as remote).

**Review by dajmcdon — CHANGES_REQUESTED** [review]
Everything else looks good modulo the Slack comment I'm about to make.

File: forecasters/zyzzyva/DESCRIPTION (hunk 1 above)
Comment (sgsmob) [reply forecasters/zyzzyva/DESCRIPTION]: done

**Comment by dajmcdon** [comment]
Sorry, I should have been more clear. You'll need to:
1. Add `# @importFrom evalcast get_target_period`
2. Add evalcast to the `Imports:` list of the DESCRIPTION.
3. Add `cmu-delphi/covidcast/R-packages/evalcast@evalcast-killcards` to the `Remotes:` list of the DESCRIPTION.

**Comment by sgsmob** [comment]
> Sorry, I should have been more clear. You'll need to:
> 
> 1. Add `# @importFrom evalcast get_target_period`
> 2. Add evalcast to the `Imports:` list of the DESCRIPTION.
> 3. Add `cmu-delphi/covidcast/R-packages/evalcast@evalcast-killcards` to the `Remotes:` list of the DESCRIPTION.

Thanks for the clarity.  I think this is fixed now.

**Review by dajmcdon — APPROVED** [review]

**Comment by dajmcdon** [comment]
I forgot to enable the CI checks before telling you to open the PR. I've approved, but I'm going to close this without merging. Can you just do the PR again?

**Pull request closed without merging**